src/daemon/custom_image_host.cpp

```diff
@@ -49,6 +49,56 @@ auto multipass_default_aliases(mp::URLDownloader* url_downloader)
 
     default_aliases.insert({"core", core_image_info});
 
+    // Image information on 16.04
+    const QString url_16{"https://cloud-images.ubuntu.com/releases/16.04/release/ubuntu-16.04-server-cloudimg-amd64-disk1.img"};
+    const auto last_modified = url_downloader_16->last_modified({url_16});
```

Comment:
`last_modified` has already been declared.  Either rename to be unique or fix the previous declaration to not be `const` and drop `auto` here.

Also in this line, `url_downloader_16` has not been declared anywhere.